core/src/calleeFunctions/OneInchCallee.ts

```diff
@@ -40,16 +40,16 @@ const getMarketPrice = async function (
     collateralAmount: BigNumber
 ): Promise<{ price: BigNumber; pools: undefined }> {
     // convert collateral into DAI
-    const collateralAmountWad = collateralAmount.shiftedBy(WAD_NUMBER_OF_DIGITS).toFixed(0);
+    const collateralAmountDecimals = collateralAmount.shiftedBy(collateral.decimals).toFixed(0);
```

Comment:
This is typically called `*IntegerAmount` in our codebase, cause it holds something opposite of decimals

```diff
- const collateralAmountDecimals = collateralAmount.shiftedBy(collateral.decimals).toFixed(0);
+ const collateralIntegerAmount = collateralAmount.shiftedBy(collateral.decimals).toFixed(0);
```